feed-integration-webapp/src/test/resources/samplePatient.json

```diff
@@ -0,0 +1,192 @@
+{
+  "uuid": "105059a8-5226-4b1f-b512-0d3ae685287d",
+  "display": "GAN200053 - Test NoOpenVisit Scenario",
+  "identifiers": [
+    {
+      "display": "Bahmni Id = GAN200053",
+      "uuid": "a005b150-a66f-4cd9-adf3-e30d5eb13153",
+      "identifier": "GAN200053",
+      "identifierType": {
+        "uuid": "81433852-3f10-11e4-adec-0800271c1b75",
+        "display": "Bahmni Id",
+        "links": [
+          {
+            "uri": "NEED-TO-CONFIGURE/ws/rest/v1/patientidentifiertype/81433852-3f10-11e4-adec-0800271c1b75",
+            "rel": "self"
+          }
+        ]
+      },
+      "location": null,
+      "preferred": true,
+      "voided": false,
+      "links": [
+        {
+          "uri": "NEED-TO-CONFIGURE/ws/rest/v1/patient/31e164fa-dd5b-4a10-a487-33e3a97e6198/identifier/a005b150-a66f-4cd9-adf3-e30d5eb13153",
+          "rel": "self"
+        },
+        {
+          "uri": "NEED-TO-CONFIGURE/ws/rest/v1/patient/31e164fa-dd5b-4a10-a487-33e3a97e6198/identifier/a005b150-a66f-4cd9-adf3-e30d5eb13153?v=full",
+          "rel": "full"
+        }
+      ],
+      "resourceVersion": "1.8"
+    }
+  ],
+  "person": {
+    "uuid": "31e164fa-dd5b-4a10-a487-33e3a97e6198",
+    "display": "Test NoOpenVisit Scenario",
```

Comment:
Is there a reason why this JSON doesn't look like an actual Patient json? It would be better to put real sounding names to ensure the test acts as actual documentation on what is expected. Having a given name with semi-colons is invalid.

Reply:
A real patient name has been added for test files.